website/src/segment.css.ts

```diff
@@ -146,6 +146,7 @@ const annotationSectionBase = style([
     width: "100%",
     position: "relative",
     marginBottom: vspace.half,
+    display: "flex",
```

Comment:
It looks like this change actually totally messed up the mobile view. Make sure to test that stuff first even if I suggest it 🤣 . A better solution might be something like this:
```ts
"@media": {
  [mediaQueries.medium]: {
    display: "flex",
    flexFlow: "row wrap",
  }
}
```